### PR TITLE
[SCB-1109] local-service-registry sample of java-chassis can not run directly

### PR DESCRIPTION
**Pull request by MabinGo**
Fix the bug that the local-service-registry sample cannot be run correctly.

Signed-off-by: MabinGo <bin.ma@huawei.com>

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: samples/local-service-registry/README.md [hunk 1]
```diff
@@ -33,13 +33,17 @@ Please find the details as follows.
...
     public class xxxClient {
       public static void main(String[] args) {
+      //"yourpath" must be used absolute path
 　　    System.setProperty("local.registry.file", "/yourpath/registry.yaml");
-        //your code
+
+      //your code
+      ......
+
 　　    System.clearProperty("local.registry.file");
```
Comment (wujimin) [inline samples/local-service-registry/README.md]: clear is not necessary?

File: samples/local-service-registry/local-registry-client/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryclient/LocalRegistryClient.java [hunk 2]
```diff
@@ -30,10 +30,14 @@
   private static RestTemplate templateNew = RestTemplateBuilder.create();
 
   public static void main(String[] args) throws Exception {
-    System.setProperty("local.registry.file", "src/main/resources/registry.yaml");
+    String absoluteFilePath = Class.class.getClass().getResource("/").getPath() + "registry.yaml";
```
Comment (wujimin) [inline samples/local-service-registry/local-registry-client/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryclient/LocalRegistryClient.java]: better to support load based on classloader
current IMPL only can load from disk, can not load from jar resource

Comment (MabinGo) [reply samples/local-service-registry/local-registry-client/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryclient/LocalRegistryClient.java]: change to use classloader to get absolute path

File: samples/local-service-registry/README.md (hunk 1 above)
Comment (wujimin) [reply samples/local-service-registry/README.md]: process will exit, clear or not, have the same effect？

File: samples/local-service-registry/local-registry-client/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryclient/LocalRegistryClient.java [hunk 3]
```diff
@@ -30,10 +30,14 @@
   private static RestTemplate templateNew = RestTemplateBuilder.create();
 
   public static void main(String[] args) throws Exception {
-    System.setProperty("local.registry.file", "src/main/resources/registry.yaml");
+    String absoluteFilePath = ClassLoader.getSystemResource("").getPath() + "registry.yaml";
```
Comment (wujimin) [inline samples/local-service-registry/local-registry-client/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryclient/LocalRegistryClient.java]: after package to jar, can not load the file
to support this feature, should modify org.apache.servicecomb.serviceregistry.client.LocalServiceRegistryClientImpl#LocalServiceRegistryClientImpl()?

**Comment by liubao68** [comment]
Can we add a new yaml configrution instead of system property? This is not quite convenient in many scenarios.

File: samples/local-service-registry/README.md (hunk 1 above)
Comment (MabinGo) [reply samples/local-service-registry/README.md]: ok, clearProperty can be deleted

File: samples/local-service-registry/local-registry-client/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryclient/LocalRegistryClient.java (hunk 3 above)
Comment (MabinGo) [reply samples/local-service-registry/local-registry-client/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryclient/LocalRegistryClient.java]: I try to modify LocalServiceRegistryClientImpl class to check whether the file exists and read the file by classloader, the file must be put in the resource directory. Test is OK, pls help me to review,thks

**Comment by MabinGo** [comment]
@liubao68 I think we can open a new issue to discuss this suggestion : )

**Pull request merged**